src/Equinor.ProCoSys.Preservation.WebApi/Seeding/Seeders.cs

```
@@ -93,14 +93,17 @@ public static void AddTags(this IEnumerable<Project> projects, int entryCountPrP
                     var tag = new Tag(
                         plant, 
                         TagType.Standard,
+                        Guid.NewGuid(),
```

not a test project, but I assume this is for test purposes of some sort.

---

As name of class say: Used for seed dummy data into database. Used early in project, and has not been used since then. (We could consider to throw away code, but it don't disturb that much either)